prac2_login_verification_UI/verification.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:flutter/material.dart';
+
+void main()=> runApp(MaterialApp(
+  home: verification(),
+));
+class verification extends StatefulWidget {
+  const verification({Key? key}) : super(key: key);
+
+  @override
+  State<verification> createState() => _verificationState();
+}
+
+class _verificationState extends State<verification> {
+  bool value=true;
+  String radio='We’ll send code to your email that you can enter to verify your identity';
+  bool isEmail=true;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Padding(

```

Comment:
Consider wrapping the body in a **SafeArea** widget.
Please refer to the documentation to get an idea of what it is and how to use it.

Reply:
Resolved in 4754d2b and 7c24e01... :+1: